app/views/shared/_navbar.html.erb

```diff
@@ -8,6 +8,7 @@
     <% end %>
     <div>
       <% if user_signed_in? %>
+        <span class="me-3"><%= link_to "New Offer", new_offer_path, class:"btn btn-primary rounded" %></span>
```

Comment:
We haven't used rounded corners so far for buttons. I think we should stick to one style.